src/components/NewPerson/NewPerson.scss

```
@@ -0,0 +1,78 @@
+.form {
+    background: #f5f5f5;
```

```suggestion
.form {
  background: #f5f5f5;
```
двух отступов достаточно я думаю